modules/renter/hostdb/scan.go

```diff
@@ -99,7 +99,7 @@ func (hdb *HostDB) decrementReliability(addr modules.NetAddress, penalty types.C
 	// Look up the entry and decrement the reliability.
 	entry, exists := hdb.allHosts[addr]
 	if !exists {
-		// TODO: should panic here
+		build.Critical("host to be decremented did not exist in hostdb")
```

Comment:
it does scream that there are problems though, with the idea that the user should make bug reports about them.

They are too common and too poorly understood to put into production. And all of our nodes usually run in DEBUG mode so that we can find the tricky stuff.